alerts/README.md

```diff
@@ -0,0 +1,50 @@
+Contains alerts templates under a single base
+
+# Available bases
+
+- common: Contains templates for alerts we deploy everywhere
+- aws: Appends common base with alerts only for aws clusters
+
+# Environment variables
+
+The following environment variables are used and expected to be patched:
+- ENVIRONMENT: exp-1|dev|prod
+- PROVIDER: aws|gcp|merit
+
+# Patch inside an init container
+
+One way to patch is using `envsubst` inside an init container in the ruler
+deployment. For example:
+
+```
+      initContainers:
+        - name: render-alerts
+          image: alpine
+          env:
+            - name: ENVIRONMENT
+              value: exp-1
+            - name: PROVIDER
+              value: aws
+          args:
+            - /bin/sh
+            - -c
+            - |
+              apk add --no-cache gettext;
+              for file in $(ls /var/thanos/rule-templates/*.tmpl); do
+                f=$(basename $file);
+                envsubst '${ENVIRONMENT},${PROVIDER}' < $file > /var/thanos/rules/${f%.*};
```

Comment:
I can be wrong, but I don't think you need `'${ENVIRONMENT},${PROVIDER}'` here, it will be automatically pulled from `env`.

Reply:
This actually tells envsubst to patch only these 2 variables and leave everything else that is behind a `$` as is. Otherwise you end up replacing things like `{{ $labels }}` with whitespaces